amundsen_application/static/js/ducks/user/api/v0.ts

```diff
@@ -13,10 +13,28 @@ export function getLoggedInUser() {
 
 export function getUserById(userId: string) {
   return axios.get(`/api/metadata/v0/user?user_id=${userId}`)
-  .then((response: AxiosResponse<UserResponse>) => {
-    return response.data.user;
-  })
-  .catch((error: AxiosError) => {
-    return {};
-  });
+    .then((response: AxiosResponse<UserResponse>) => {
+      return response.data.user;
+    })
+    .catch((error: AxiosError) => {
+      return {};
+    });
+}
+
+export function getUserOwn(userId: string) {
+  return axios.get(`/api/metadata/v0/user/own?user_id=${userId}`)
+    .then((response: AxiosResponse<UserOwnResponse>) => {
+      return response.data
```

Comment:
Do we have type safety here, I've assumed yes but recently I haven't been so sure with where TypeScript succeeds with its inferences versus not. In `ducks/user/sagas.ts` if we try to access `userOwn.notValid` do we get a type error?

Reply:
Doesn't seem like it-- probably because TS can't infer the type based on the `catch` part of these. Will investigate to see if there's a better pattern.

Reply:
Yeah I'll capture a ticket for this.